### PR TITLE
Improve model performances on heart disease.

### PR DESCRIPTION
**Pull request by fd0r**
- Change optimizer from SGD to Adam
- Change loss from MSE to BinaryCrossEntropy(with_logits=True)

### CODE REVIEW

**Comment by fd0r** [comment]
With those changes the final test accuracy is improved from `53.333%` to `93.333%`.

**Comment by dubek** [comment]
Thanks @fd0r for your contributions. We're looking into it.
